src/pages/BurgerOrder/components/BurgerBuilder/components/SauceBuildContainer/SauceBuildContainer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  addBurgerSauce,
+  removeBurgerSauce,
+} from '../../../../../../common/store/customBurgerStore/customBurgerActionCreators';
+import { Sauce } from '../../../../../../common/models/customBurgerTypes';
+import { Box, Button } from '@material-ui/core';
+import './sauceBuildContainer.scss';
+
+type SauceTypes = {
+  sauce: Sauce[];
+};
+
+const SauceBuildContainer = ({ sauce }: SauceTypes): JSX.Element => {
+  const dispatch = useDispatch();
+
+  const handleAddSauce = (e: React.MouseEvent, sauceName: string) => {
+    e.preventDefault();
```

Comment:
Material UI Button component should have already handled the default mouse event nicely